Files/ViewModels/SettingsViewModels/AboutViewModel.cs

```diff
@@ -57,7 +57,7 @@ private async void ClickAboutFeedbackItem(ItemClickEventArgs e)
                     break;
 
                 case "Documentation":
-                    await Launcher.LaunchUriAsync(new Uri(@"https://files-community.github.io/docs"));
+                    await Launcher.LaunchUriAsync(new Uri(@"https://files.community/docs"));
```

Comment:
Should this be a constant to make it easier to update in the future?

Reply:
Sounds like a good idea, where would you store the constant?